_rules/SC1-1-1-image-has-name.md

```diff
@@ -19,21 +19,17 @@ authors:
 
 ### Applicability
 
-The rule applies to any HTML element with the [semantic role][] of `image` that is [exposed to assistive technologies][].
-
-**NOTE**: Any image marked as decorative, by using either `role="presentation"`, `role="none"` or an empty `alt` attribute in combination with either no `title` attribute or an empty `title` attribute, does not have the role of `image` according to [Accessible Rich Internet Applications (WAI-ARIA) 1.1](https://www.w3.org/TR/wai-aria-1.1/)
+The rule applies to HTML `img` elements or any HTML element with the [semantic role](#semantic-role) of `img` that is exposed to assistive technologies.
 
 ### Expectation
 
-Each target element has an [accessible name][] that is [non-empty][].
-
-## Assumptions
+Each target element has an accessible name that is non-empty or is marked as decorative.
```

Comment:
Link to definitions of accessible name, non-empty and decorative

Reply:
Updated, please review again.